### PR TITLE
chore: remove useless TelegrafRequestPlugin

### PR DESCRIPTION
**Pull request by bednar**
The `TelegrafRequestPlugin` is not used in any operation, so it looks like that is useless.

### CODE REVIEW

File: src/common/schemas/TelegrafRequestPlugin.yml [hunk 1]
```diff
@@ -1,14 +0,0 @@
...
-    - $ref: "./TelegrafPluginInputDocker.yml"
-    - $ref: "./TelegrafPluginInputFile.yml"
-    - $ref: "./TelegrafPluginInputKubernetes.yml"
-    - $ref: "./TelegrafPluginInputLogParser.yml"
-    - $ref: "./TelegrafPluginInputProcstat.yml"
-    - $ref: "./TelegrafPluginInputPrometheus.yml"
-    - $ref: "./TelegrafPluginInputRedis.yml"
-    - $ref: "./TelegrafPluginInputSyslog.yml"
-    - $ref: "./TelegrafPluginOutputFile.yml"
-    - $ref: "./TelegrafPluginOutputInfluxDBV2.yml"
```
Comment (glinton) [inline src/common/schemas/TelegrafRequestPlugin.yml:14]: Want to remove these files as well, if possible?

Comment (bednar) [reply src/common/schemas/TelegrafRequestPlugin.yml:14]: Yes, you are right. PR is updated.

Comment (bednar) [reply src/common/schemas/TelegrafRequestPlugin.yml:14]: Sorry for misunderstood. Files are removed.

**Review by glinton — APPROVED** [review]
thanks

**Pull request merged**